### PR TITLE
Add more information on how install and configure the FOSUserBundle

### PR DESCRIPTION
**Pull request by piotaixr**
Following as in #242, i was first very confused on how to enable FOSUB in the API Platform Standard Edition.

Here is the information i would have liked to have when reading the documentation.
Hope it helps other people!

Edit: Question: Is enabling the form component the correct way of doing things?

### CODE REVIEW

**Comment by soyuka** [comment]
> Edit: Question: Is enabling the form component the correct way of doing things?

Not necessarily if you use directly the `jwt` route to login and get a token for example.

File: core/fosuser-bundle.md [hunk 1]
```diff
@@ -3,6 +3,31 @@
...
+
+If you are using the API Platform Standard Edition, you will need to enable the form services in the symfony framework configuration options:
+
+```yaml
+# app/config/config.yml
+framework:
+    form: { enabled: true }
+```
+
+## Enable the bridge
```
Comment (meyerbaptiste) [inline core/fosuser-bundle.md]: `Enabling` to keep consistency with the others titles

File: core/fosuser-bundle.md [hunk 2]
```diff
@@ -3,6 +3,31 @@
 API Platform Core is shipped with a bridge for [FOSUserBundle](https://github.com/FriendsOfSymfony/FOSUserBundle). If the
 FOSUser bundle is enabled, this bridge will use its `UserManager` to create, update and delete user resources.
 
+## Installing the bundle
```
Comment (meyerbaptiste) [inline core/fosuser-bundle.md]: You have to add these chapters in the `index.md` file.

**Comment by meyerbaptiste** [comment]
Thanks @piotaixr

**Pull request merged**